systems-management/nisysmgmt.yml

```diff
@@ -0,0 +1,2030 @@
+      replaces:
+        type: array
+        description: An array containing information about other packages this package replaces.
+        items:
+          type: string
+          description: The name of a package that this package replaces. This also contains information about the version of the package.
+          example: "b235b862-6a92-4a04-a8b2-6d71f777de67 (< 16.99.65535)"       
+      section:
+        type: string
+        description: The section of the package.
+        example: Infrastructure
+      size:
+        type: integer
+        x-ni-data-type: I64
+        description: The size of the package in bytes.
+        example: 750070
+      source:
+        type: string
+        description: The source of this package, if any.
+        example: "thunar-archive-plugin_0.3.1.bb"
+      storeproduct:
+        type: boolean
+        description: Whether the package represent a product in the Store or not.
+        example: true
+      suggests:
+        type: array
+        description: An array containing information about other packages this package suggests.
+        items:
+          type: string
+          description: The name of a package that this package suggests. This also contains information about the version of the package.
+          example: "ni-systemlink-client-2018.5-realtime-bin (>= 18.5.0)"       
+      tags:
+        type: string
+        description: Tags of the package.
+        example: "KillBits, Security Update 2013"    
+      uservisible:
+        type: boolean
+        description: The installed packages with this attribute set to true are considered to be installed by a user, thus not installed to fulfill some dependency. This is needed input for the calculation of unneeded packages when removing a package.
+        example: true
+      version:
+        type: string
+        description: The version of the package.
+        example: "2.0.0.22-0+d22"
+    required:
+      - name
+      - version
+
+  Error:
+    description: Contains error information
+    type: object
+    properties:
+      name:
+        description: The string error code.
+        type: string
+        example: InvalidQuerySyntax
+      code:
+        description: Numeric error code.
+        type: integer
+        example: -254004
+      resourceType:
+        description: The type of resource associated with the error.
+        type: string
+        example: System
+      resourceId:
+        description: The identifier of the resource associated with the error.
+        type: string
+        example: VMware_Virtual_Platform--SN-VMware-56_4d_d2_08_ce_89_1b_70-e6_32_fc_21_6c_d0_d9_f9--MAC-00-0C-29-D0-D9-F9
+      message:
+        description: The filled in error message.
+        type: string
+        example: The provided query is invalid. 
+      args:
+        description: Positional argument values for the error code.
+        type: array
+        items:
+          type: string
+          example: invalidquery
```

Comment:
'invalidquery' should not be in the args array.  The args array is for message inserts.